manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml

```diff
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.5-1-22621-608.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SecombaGmbH.Boxcryptor
+PackageVersion: 2.55.2774
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+  Custom: REBOOT=ReallySuppress
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+FileExtensions:
+- bc
+- bch
+- bckey
+- bclink
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.EdgeWebView2Runtimes
```

Comment:
```suggestion
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```

Reply:
Sorry for that! I happened to copy/paste the wrong lines when testing the validation system.